tdrs-backend/tdpservice/email/helpers/admin_notifications.py

```diff
@@ -33,3 +33,37 @@ def email_admin_deactivated_user(user):
             text_message=text_message,
             logger_context=logger_context
         )
+
+def email_system_owner_system_admin_role_change(user, action):
+    """Send an email to the System Owner when a user is assigned or removed from the System Admin role."""
+    from tdpservice.users.models import User
+    from tdpservice.email.email_enums import EmailType
+    from tdpservice.email.email import automated_email, log
+    from tdpservice.email.tasks import get_system_owner_email
```

Comment:
is there a reason for the imports in the function? i see it in other functions in this file, so just curious

Reply:
I think originally I got circular error, but will check back again because I changed things around

Reply:
Still getting circular imports when put at the top?

Reply:
Yes, but looks like it is because of ```get_ofa_admin_user_emails```.